app/views/ldap_auth_sources/_form.html.erb

```diff
@@ -43,7 +43,17 @@ See COPYRIGHT and LICENSE files for more details.
   </div>
 <% end %>
 
-<section class="form--section">
+<% if @ldap_auth_source.seeded_from_env? %>
+  <div class="op-toast -warning">
+    <div class="op-toast--content">
+      <%= t(:label_seeded_from_env_warning) %>
+      <br/>
+      <%= link_to t('ldap_auth_sources.back_to_index'), { action: :index } %>
+    </div>
+  </div>
+<% end %>
+
+<%= content_tag :fieldset, class: 'form--fieldset', disabled: @ldap_auth_source.seeded_from_env? do %>
```

Comment:
This was necessary to easily block the entire form using `disabled` on the fieldset (which avoids having to do it on a per-form-field level)